src/modules/permissions/PermissionsFilters.tsx

```diff
@@ -79,6 +79,7 @@ export const ClientPermissionsFilter: React.FC<
 > = ({ id, ...props }) => {
   const { data, loading } = useGetClientPermissionsQuery({
     variables: { id },
+    skip: !id,
```

Comment:
When is this happening? It looks like a case of us expecting `clientId` to be present in the route, but not finding it?